webapp/components/Editor/SuggestionList.vue

```diff
@@ -7,15 +7,15 @@
       :class="{ 'is-selected': navigatedItemIndex === index }"
       @click="selectItem(item)"
     >
-      {{ createItemLabel(item) }}
+      {{ createItemLabel(item) | truncate(50) }}
     </li>
     <template v-if="isHashtag">
-      <li v-if="!query" class="suggestion-list__item hint">
-        {{ $t('editor.hashtag.addLetter') }}
-      </li>
+      <li v-if="!query" class="suggestion-list__item hint">{{ $t('editor.hashtag.addLetter') }}</li>
       <template v-else-if="!filteredItems.find(el => el.id === query)">
         <li class="suggestion-list__item hint">{{ $t('editor.hashtag.addHashtag') }}</li>
-        <li class="suggestion-list__item" @click="selectItem({ id: query })">#{{ query }}</li>
+        <li class="suggestion-list__item" @click="selectItem({ id: query })">
+          #{{ query | truncate(50) }}
```

Comment:
> [<img alt="alina-beck" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/alina-beck) **Authored by [alina-beck](https://github.com/alina-beck)**
_<time datetime="2019-09-03T14:20:52Z" title="Tuesday, September 3rd 2019, 4:20:52 pm +02:00">Sep 3, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -7,15 +7,15 @@
       :class="{ 'is-selected': navigatedItemIndex === index }"
       @click="selectItem(item)"
     >
-      {{ createItemLabel(item) }}
+      {{ createItemLabel(item) | truncate(50) }}
     </li>
     <template v-if="isHashtag">
-      <li v-if="!query" class="suggestion-list__item hint">
-        {{ $t('editor.hashtag.addLetter') }}
-      </li>
+      <li v-if="!query" class="suggestion-list__item hint">{{ $t('editor.hashtag.addLetter') }}</li>
       <template v-else-if="!filteredItems.find(el => el.id === query)">
         <li class="suggestion-list__item hint">{{ $t('editor.hashtag.addHashtag') }}</li>
-        <li class="suggestion-list__item" @click="selectItem({ id: query })">#{{ query }}</li>
+        <li class="suggestion-list__item" @click="selectItem({ id: query })">
+          #{{ query | truncate(50) }}
```


:+1:

I just had to google what this does – very confusing if you don't know about Vue.js filters... Great opportunity to learn about them, though! So thanks for this! :smile:

Reply:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-09-04T12:19:45Z" title="Wednesday, September 4th 2019, 2:19:45 pm +02:00">Sep 4, 2019</time>_

---


@mattwr18 and I learned this by @roschaefer ! 😍💫